src/swarm-tui/Swarm/TUI/Model/UI.hs

```diff
@@ -8,52 +8,14 @@
 -- SPDX-License-Identifier: BSD-3-Clause
 module Swarm.TUI.Model.UI (
```

Comment:
We don't follow this consistently (yet), but it might be nice to re-export the whole `Swarm.TUI.Model.UI.Gameplay` here, so that there are fewer changes in the imports.

It's a drop of water in the ocean though, so feel free to ignore this suggestion.